src/driver/oracle/OracleDriver.ts

```diff
@@ -339,15 +339,15 @@ export class OracleDriver implements Driver {
      * Escapes a column name.
      */
     escape(columnName: string): string {
-        return `"${columnName}"`;
+        return columnName;
     }
```

Comment:
Even if this isn't technically needed, does it hurt things to have it?

Reply:
Hey, 

As part of the changes needed, if I was to use a package in the FROM statement, I can't have quotes around it or it throws an ORA exception. This was a first attempt at removing the quotes (since Oracle doesn't need them) but - as discovered - it does it for more than just table names, also column, aliases, etc from what I seen. 

I am working on a better solution for this